metrics/src/main/java/com/expedia/adaptivealerting/metrics/functions/source/graphite/Datapoint.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2018-2019 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expedia.adaptivealerting.metrics.functions.source.graphite;
+
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.Data;
+
+@JsonSerialize
+@Data
+public class Datapoint {
+    private double value;
+
+    private long timestamp;
+
+    public Datapoint(double value, long timestamp){
+        this.value = value;
+        this.timestamp = timestamp;
+    }
```

Comment:
Note that you can use `@AllArgsConstructor` here. Not required but it's an option.

Reply:
Fixed.